src/docs/user.yaml

```diff
@@ -0,0 +1,26 @@
+paths:
+  /api/user/{id}:
+    get:
+      tags:
+        - get users
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          required: true
+          description: 'ID of the user'
+          examples:
+            'real ID of a deleted user':
+              value: '62ee5c280262c62f56124dfd'
+            'custom':
+              value: ''
+      responses:
+        '200':
+          description: 'OK (shows user)'
+        '404':
+          description: 'Not Found (user was not found)'
+        '401':
+          description: 'Unauthorized (one needs to be logged in)'
+        '500':
+          description: 'Internal server error (ID is not a valid ObjectId)'
```

Comment:
I am not sure if the internal server error translates to ID is not a valid ObjectId

Reply:
It's not, but I also got an `ObjectId` error. It was a 500, if I remember correctly. The error was about the ObjectId not being a "real" ObjectId.

Reply:
Then you can specify this could be the issue. Because it might get the `500` for other reasons

Reply:
Added it to description.